05_matplotlib.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Formative Assessment 1:</p><p>What is the correct way to change the label for the x-axis? / ¿Cuál es la forma correcta de cambiar la etiqueta del eje x?</p><p> </p><p>plt.label('change name')</p><p>plt.xlabel('change name')</p><p>plt.xlabel(change name)</p><p>plt.label(*change name*)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>